unitex/src/fr/umlv/unitex/Unitex.java

```diff
@@ -60,14 +60,18 @@ public static boolean isRunning() {
 	
 	public static void main(final String[] args) {
 		running=true;
-		EventQueue.invokeLater(new Runnable() {
-			@Override
-			public void run() {
-				launchUnitex(args);
-			}
-		});
+
+		if (!Config.isRunning(args)) {
+			EventQueue.invokeLater(new Runnable() {
+				@Override
+				public void run() {
+					launchUnitex(args);
```

Comment:
Launching Gramlab.jar (not Unitex.jar) within the argument `-Dunitex.binary.dir` doesn't work, the argument isn't recognized:

```
java -jar Gramlab.jar -Dunitex.binary.dir="/home/user/Unitex-GramLab-3.1/App"
```

![gramlab_binary_dir_parsing_error](https://cloud.githubusercontent.com/assets/800627/23075615/7823e2d4-f53d-11e6-84db-103627359de3.png)